docs/_spec/public/scripts/main.js

```diff
@@ -1,44 +1,38 @@
 function currentChapter() {
-  var path = document.location.pathname;
-  var idx  = path.lastIndexOf("/") + 1;
-  var chap = path.substring(idx, idx + 2);
-  return parseInt(chap, 10);
+  return parseInt(document.location.pathname.split('/').pop().substr(0, 2), 10);
 }
 
 function heading(i, heading, $heading) {
-  var currentLevel = parseInt(heading.tagName.substring(1));
-  var result = "";
+  const currentLevel = parseInt(heading.tagName.substring(1));
+  const headerCounts = this.headerCounts;
```

Comment:
Can we go back to using `this.headerCounts` directly (without assigning to const)? It reads a bit confusing here since we later modify its contents